src/pages/dmt5/_download-app.tsx

```diff
@@ -1,20 +1,36 @@
 import React from 'react'
 import styled from 'styled-components'
 import { graphql, useStaticQuery } from 'gatsby'
-import { Flex, SectionContainer, Box, Desktop, Mobile } from 'components/containers'
-import { Header, QueryImage, Text } from 'components/elements'
+import { Flex, Desktop, Mobile } from 'components/containers'
+import { Header, QueryImage } from 'components/elements'
 import { localize, Localize, LocalizedLink } from 'components/localization'
-import AppStore from 'images/svg/dmt5/app-store.svg'
-import GooglePlay from 'images/svg/dmt5/google-play.svg'
-import GooglePlayMobile from 'images/svg/dmt5/google-play-mobile.svg'
-import MacOS from 'images/svg/dmt5/macos.svg'
-import MoreInfo from 'images/svg/dmt5/more-info.svg'
-import Windows from 'images/svg/dmt5/windows.svg'
+import MoreInfo from 'images/svg/dmt5/more_info.svg'
+import MoreInfoDesktop from 'images/svg/dmt5/more-info.svg'
 import device from 'themes/device'
-import { dmt5_macos_url, dmt5_windows_url, dmt5_android_url, dmt5_ios_url } from 'common/constants'
+import LinuxOs from 'images/svg/dmt5/dmt5_linux.svg'
+import WebBrowser from 'images/svg/dmt5/dmt5_web_browser.svg'
+import Windows from 'images/svg/dmt5/dmt5_windows.svg'
+import MacOs from 'images/svg/dmt5/dmt5_macos_dmg.svg'
+import GooglePlay from 'images/svg/dmt5/dmt5_google_play.svg'
+import AppGallery from 'images/svg/dmt5/dmt5_app_gallery.svg'
+import AppStore from 'images/svg/dmt5/dmt5_app_store.svg'
+import {
+    dmt5_web_browser_url,
+    dmt5_windows,
+    dmt5_macos_url,
+    dmt5_linux_url,
+    dmt5_app_store_url,
+    dmt5_android_url,
+    dmt5_app_gallery_url,
+} from 'common/constants'
 
 type ImageWrapperProps = {
-    mt?: string
+    mt: string
+}
+type LinkType = {
+    mt: string
+    mb: string
+    mr: string
 }
```

Comment:
Please make this property optional 🙏 
When we declare it required, we will catch errors in components if all of them are not present.